scripts/commands/wizard/utils.js

```diff
@@ -57,22 +61,44 @@ async function selectTutorial (newItemType, { createTutorial, createResource, cr
         {
           type: 'list',
           name: 'tutorialId',
-          message: `Which of these existing tutorials should we add your ${newItemType} to?`,
+          message: `Which of these existing tutorials should I add your ${newItemType} to?`,
           choices: tutorialChoices.reverse()
         }
       ])
     // set data based on other selected tutorial
     if (tutorialResponses2.tutorialId !== 'new') {
       tutorial = tutorials[tutorialResponses2.tutorialId]
     // create new tutorial and set data accordingly
+    } else if (newItemType === 'quiz') {
+      log.info("you want a quiz but you're making a new tutorial that won't have any lessons in it")
+      tutorial = await createTutorial({ createLesson, createResource, createQuiz }, { skipPromptLesson: false })
     } else {
-      tutorial = await createTutorial({ createLesson, createResource }, { skipPromptLesson: true })
+      tutorial = await createTutorial({ createLesson, createResource, createQuiz }, { skipPromptLesson: true })
     }
   }
 
   return tutorial
 }
 
+async function selectLesson (tutorial) {
```

Comment:
Maybe we should rename it to `selectMultipleChoiceLesson`?
The `selectTutorial` function only selects one tutorial and does not discriminate between the tutorial types.